Mindbox.ExceptionsHandling.Tests/GraphQLLogErrorFilterTests.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using HotChocolate;
+using Microsoft.Extensions.Logging;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Mindbox.ExceptionsHandling.GraphQL;
+using Moq;
+
+namespace Mindbox.ExceptionsHandling.Tests;
+
+[TestClass]
+public class GraphQLLogErrorFilterTests
+{
+	private static readonly LogLevel _matchedLogLevel = LogLevel.Critical;
+	private readonly LoggerStub _loggerStub = new();
+
+	[TestMethod]
+	public void OnError_WithException_MatchedLogLevel()
+	{
+		var initialError = new Error("SomeMessage", exception: new Exception("MessageFromException"));
+
+		OnError(initialError);
+
+		var (logLevel, exception, message) = _loggerStub.GetSingleLoggedMessage();
+		Assert.AreEqual(_matchedLogLevel, logLevel);
+		Assert.AreEqual(initialError.Exception, exception);
+		Assert.AreEqual("MessageFromException", message);
+	}
+
+	[TestMethod]
+	public void OnError_WithSerializationException_Error()
+	{
+		var initialError = new Error(
+			"SomeMessage",
+			exception: new HotChocolate.Types.SerializationException(
+				"MessageFromException",
+				new Mock<HotChocolate.Types.IType>().Object));
+
+		OnError(initialError);
+
+		var (logLevel, exception, message) = _loggerStub.GetSingleLoggedMessage();
+		Assert.AreEqual(LogLevel.Error, logLevel);
+		Assert.AreEqual(null, exception);
+		Assert.AreEqual("SomeMessage", message);
+	}
+
+	[TestMethod]
+	public void OnError_WithoutException_Error()
```

Comment:
тут Error в двух контекстах употребляется, сложно было осознать

```suggestion
	public void OnError_WithSerializationException_ErrorLogLevel()
	{
		var initialError = new Error(
			"SomeMessage",
			exception: new HotChocolate.Types.SerializationException(
				"MessageFromException",
				new Mock<HotChocolate.Types.IType>().Object));

		OnError(initialError);

		var (logLevel, exception, message) = _loggerStub.GetSingleLoggedMessage();
		Assert.AreEqual(LogLevel.Error, logLevel);
		Assert.AreEqual(null, exception);
		Assert.AreEqual("SomeMessage", message);
	}

	[TestMethod]
	public void OnError_WithoutException_ErrorLogLevel()
```